packages/react/src/River/River.module.css

```diff
@@ -1,8 +1,9 @@
 .River {
   display: grid;
-  gap: var(--brand-River-gap-column-medium);
+  gap: var(--brand-River-small-gap);
```

Comment:
could we call this token xsmall? as 34rem is our small breakpoint

Reply:
using responsive values as we did in `Grid` instead